third_party/VENDOR-LICENSE

```diff
@@ -3560,6 +3560,213 @@ Import: github.com/knative/serving/vendor/github.com/knative/build
 
 
 
+===========================================================
```

Comment:
I haven't noticed this file before - what updates it?

Reply:
`./hack/update-deps.sh` runs a tool I wrote to collect these from vendor.  We also have presubmit checks based on the same tool (which wraps https://github.com/google/licenseclassifier) to check for forbidden licenses like WTFPL.